packages/top-nav/src/top-nav-item.css

```diff
@@ -27,8 +27,11 @@ a:focus {
 
 :host(:focus-within):before {
     border-color: var(
-        --spectrum-tabs-m-focus-ring-color,
-        var(--spectrum-alias-border-color-key-focus)
+        --highcontrast-tabs-focus-indicator-color,
+        var(
+            --spectrum-tabs-m-focus-ring-color,
+            var(--spectrum-alias-border-color-key-focus)
```

Comment:
Are these current to the new Tabs values?

Reply:
I mis-read! Internal to the repo, the dependencies always resolve to what's in the current branch, so you shouldn't need a new release to make this update. If it gives you trouble, that'll be important to know as it points to other issues.

Reply:
Ok this has been updated! For some reason I tried this initially and it didn't work but looks like it's good now